instrumentation/play/play-mvc/play-mvc-2.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_6/Play26Singletons.java

```diff
@@ -52,12 +54,17 @@ public static Instrumenter<Void, Void> instrumenter() {
     return INSTRUMENTER;
   }
 
-  public static void updateSpanNames(Context context, Request<?> request) {
+  public static void updateContext(Context context, Request<?> request) {
     String route = getRoute(request);
     if (route == null) {
       return;
     }
 
+    updateSpanNameByRoute(context, route);
+    updateHttpRoute(context, route);
+  }
+
+  private static void updateSpanNameByRoute(Context context, String route) {
     Span.fromContext(context).updateName(route);
     // set the span name on the upstream akka/netty span
     Span serverSpan = LocalRootSpan.fromContextOrNull(context);
```

Comment:
Can you remove the code that handles the `serverSpan` below? The `HttpRouteHolder` does that already

Reply:
Thank you for pointing out it. I removed them at [ff97ca1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7801/commits/ff97ca15289a0cf8969bc5a02d1c7a9e667d0f08).